hyperapp.d.ts

```diff
@@ -0,0 +1,40 @@
+export as namespace hyperapp
+
+// VirtualNode part
+
+export interface VirtualNode<Data> {
+  tag: string
+  data?: Data
+  children: VirtualElement<{}>[]
+}
+
+export type VirtualElement<Data> = VirtualNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VirtualElement<{}>[]): VirtualNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VirtualElement<{}>[] | VirtualElement<{}> | number
```

Comment:
Correct, it is. Screenshot below shows an example of what a user would see for the argument spread version of h:

![image](https://user-images.githubusercontent.com/459878/28508492-289335c0-6ff0-11e7-89b8-58fec502c245.png)

This screenshot shows an example of what a user would see for the array version of h:

![image](https://user-images.githubusercontent.com/459878/28508532-7303c7f0-6ff0-11e7-9efc-9e2c88bfaa3d.png)